docs/getting_oss_rules_of_play_rating.md

```diff
@@ -79,11 +83,11 @@ If everything is correctly set up, the output is going to look like the followin
 
 ```
 [+] Okay, we have a GitHub token, let's try to use it
-[+] The default value cache doesn't exist yet: .fosstars/github_project_value_cache.json
```

Comment:
It looks like you ran the build with already cached values. Could you please remove this file and the cache, Then check if anything else changes?

Reply:
I tried that locally but nothing changed, 
I got the same score and the same tests were failing on the local system.
The same tests btw are also failing on the upstream project as well.